src/main/java/net/pms/formats/OGG.java

```diff
@@ -0,0 +1,27 @@
+package net.pms.formats;
+
+public class OGG extends Format {
+
+	@Override
+	public Identifier getIdentifier() {
+		return Identifier.OGG;
+	}
+
+	@Override
+	public boolean transcodable() {
+		return true;
+	}
+
+	public OGG() {
+		type = VIDEO;
+	}
+
+	@Override
+	public String[] getSupportedExtensions() {
+		return new String[] {
+			"ogg",
+			"ogm",
+			"ogv"
```

Comment:
@Sami32 
> That's said, i think that you're dreaming about AAC(M4A) support fix, as for me most of the users loose their ability to play it on their physical renderer anymore.

I didn't understand that. Do you still think that AAC support is more broken than it used to be?